src/app/shared/directives/percentage-mask.directive.ts

```diff
@@ -0,0 +1,28 @@
+import {Directive, forwardRef, OnInit} from '@angular/core'
+import {PercentageMaskConfig, PercentageMaskDirective as NGXDirective, PercentageMaskInputMode} from 'fx-percentage'
+import {NG_VALUE_ACCESSOR} from '@angular/forms'
+
+export const PERCENTAGEMASKDIRECTIVE_VALUE_ACCESSOR: any = {
+  provide: NG_VALUE_ACCESSOR,
+  useExisting: forwardRef(() => PercentageMaskDirective),
+  multi: true,
+}
+
+@Directive({
+  selector: '[appPercentageMask]',
+  providers: [PERCENTAGEMASKDIRECTIVE_VALUE_ACCESSOR],
+})
+export class PercentageMaskDirective extends NGXDirective implements OnInit {
+  public optionsTemplate: PercentageMaskConfig = {
+    align: 'right',
+    allowNegative: false,
+    allowZero: true,
+    decimal: ".",
+    precision: 0,
+    prefix: '',
+    suffix: '%',
+    thousands: ',',
+    nullable: false,
+    inputMode: PercentageMaskInputMode.NATURAL,
+  }
+}
```

Comment:
Do we reaaally need this custom directive and additional fx-percentage for showing percentages? What are the benefits of using this dependency over a conventional percent pipe included in Angular by default?

Reply:
Ah, sorry, if this is for input, then ok. btw. do decimals work for you? I cannot write comma or dot to set decimals.

Reply:
I set decimal precision to 0 - so you can only set 1%, 2%, etc. How many decimal places do we need to display?

Reply:
Ah, no. It's ok for now.